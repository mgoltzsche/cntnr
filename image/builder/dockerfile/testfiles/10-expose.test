FROM alpine:3.7
EXPOSE 80 443/tcp

# ASSERT CFG /config/ExposedPorts/443~1tcp=map[]
# ASSERT CFG /config/ExposedPorts/80=map[]