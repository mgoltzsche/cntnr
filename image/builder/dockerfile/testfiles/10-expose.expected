FROM alpine:3.7
EXPOSE 80 443/tcp